golang-github-vbatts-tar-split/golang-github-vbatts-tar-split.spec

```diff
@@ -0,0 +1,177 @@
+%if 0%{?fedora} || 0%{?rhel} == 6
+%global with_devel 1
+%global with_bundled 0
+%global with_debug 1
+%global with_check 0
+%global with_unit_test 1
+%else
+%global with_devel 1
+%global with_bundled 0
+%global with_debug 1
+%global with_check 0
+%global with_unit_test 1
+%endif
+
+%if 0%{?with_debug}
+%global _dwz_low_mem_die_limit 0
+%else
+%global debug_package   %{nil}
+%endif
+
+%global provider        github
+%global provider_tld    com
+%global project         vbatts
+%global repo            tar-split
+# https://github.com/vbatts/tar-split
+%global provider_prefix %{provider}.%{provider_tld}/%{project}/%{repo}
+%global import_path     %{provider_prefix}
+%global commit          d50e5c9283da469398d84078519de569f617be6f
+%global shortcommit     %(c=%{commit}; echo ${c:0:7})
+
+Name:           golang-%{provider}-%{project}-%{repo}
+Version:        0.9.12
+Release:        1%{?dist}
+Summary:        tar archive assembly/disassembly
+License:        BSD 3-clause
+URL:            https://%{provider_prefix}
+Source0:        https://%{provider_prefix}/archive/%{commit}/%{repo}-%{shortcommit}.tar.gz
+
+# e.g. el6 has ppc64 arch without gcc-go, so EA tag is required
+ExclusiveArch:  %{?go_arches:%{go_arches}}%{!?go_arches:%{ix86} x86_64 %{arm}}
+# If go_compiler is not set to 1, there is no virtual provide. Use golang instead.
+BuildRequires:  %{?go_compiler:compiler(go-compiler)}%{!?go_compiler:golang}
+
+%description
+Pristinely disassembling a tar archive, and stashing needed raw bytes and offsets to reassemble a validating original archive.
+
+%if 0%{?with_devel}
+%package devel
+Summary:       %{summary}
+BuildArch:     noarch
+
+%if 0%{?with_check} && ! 0%{?with_bundled}
+%endif
+
+%if ! 0%{?with_bundled}
+BuildRequires: golang(github.com/Sirupsen/logrus)
+BuildRequires: golang(github.com/codegangsta/cli)
+%endif
+
+Provides:      golang(%{import_path}/archive/tar) = %{version}-%{release}
+Provides:      golang(%{import_path}/tar/asm) = %{version}-%{release}
+Provides:      golang(%{import_path}/tar/storage) = %{version}-%{release}
+Provides:      golang(%{import_path}/cmd/tar-split) = %{version}-%{release}
```

Comment:
cmd/tar-split is not importable. tar-split contains only files with 'package name' directive.
What is missing is golang(%{import_path}/version)